enterprise/com/src/main/java/org/neo4j/com/Client.java

```diff
@@ -208,7 +208,7 @@ public void start()
             protected ChannelContext create()
             {
                 msgLog.info( threadInfo() + "Trying to open a new channel from " + origin + " to " + destination,
-                        true );
+                        Boolean.TRUE );
```

Comment:
we should be able to just remove those true's from log messages in this and other classes that do the same thing